tests/Tests/Unit/ArrayTraitTest.php

```diff
@@ -50,7 +50,7 @@ public function testPluck()
         $value = '123';
         $key = 'key';
         $array = [$key => $value];
-        $actualValue = $this->implementation->call('pluck', [$key, &$array]);
+        $actualValue = $this->implementation->pluck($key, $array);
```

Comment:
```suggestion
        $actualValue = $this->implementation->pluck($key, &$array);
```

Reply:
It's not necessary to supply the `&` on this side, since it's part of the `pluck` method signature already

Reply:
More information: https://www.php.net/manual/en/language.references.pass.php

> Note: There is no reference sign on a function call - only on function definitions. Function definitions alone are enough to correctly pass the argument by reference.

And here as well: https://stackoverflow.com/questions/8971261/php-5-4-call-time-pass-by-reference-easy-fix-available

Call-time pass-by-reference was deprecated in PHP 5.3 and removed entirely in PHP 5.4